### ISSUE TITLE
Input for Active Learning and Reinforcement; Adding new annotations after manually refinement

### ISSUE THREAD

**Issue by aidanpriceUON** [YttriLab/A-SOID#83]
I'm attempting to build a model capable of tracking grooming and rearing of a mouse from a bottom-up perspective. I noticed in the reinforcement step that it was exclusively through selected low-confidence labels that I could manually annotate. I've got a relatively large chunk of annotated Boris files, and I was wondering if there was any way to add additional Boris annotations past the initial file to reinforce the model.

**Comment by JensBlack** [comment]
Hi @aidanpriceUON,

The best way to do this, is to actually put the additional data in as training data in the first step.
Under the hood, this algorithm is not able to go through multiple training steps, but a new instance is trained from scratch with the refined training set after each iteration. The great thing about A-SOiD is that training is really fast, so don't hesitate to restart a project. Unfortunately, you won't be able to take any manually refinement progress with you.

So what I would do is:
1. Add all available annotated data to a new project
2. Go through the feature extraction and active learning step again.
--> This should considerably improve your models performance.

---
Let me know if this helped and feel free to close the issue
